ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py

```diff
@@ -44,3 +44,48 @@ def tan(
 
 
 tan.unsupported_dtypes = {"torch": ("float16",)}
+
+
+def cosh(
```

Comment:
Please revert all changes made to this file as it's not related to the function you're working on